eng/common/scripts/apiview-helper.ps1

```diff
@@ -0,0 +1,56 @@
+
+
+function Map-LanguageToAPiview-Filter($language)
```

Comment:
As mentioned I'm OK with this temporarily but I'd like to eliminate the need to do this mapping in a central place as it will hurt adding support for other languages independent of making core changes. Please file an follow item for it.

Reply:
Filed and assigned issue to myself. https://github.com/Azure/azure-sdk-tools/issues/1625